packages/rum/src/rum.ts

```diff
@@ -88,6 +106,7 @@ export interface RumResourceEvent {
     kind: ResourceKind
   }
   traceId?: number
+  userActionId?: string
```

Comment:
Since this attribute does not exist yet, I wonder if it would not be preferable to store it as `userAction.id` in order to be aligned with future format

Reply:
This attribute is the one specified in the "User actions data format". What do you mean by "future format"?

Reply:
actually, there is still some discussion on the naming, let's stick with this one for now